Cinnamenu@json/files/Cinnamenu@json/applet.js

```diff
@@ -454,6 +462,7 @@ CinnamenuApplet.prototype = {
 
   on_applet_removed_from_panel: function() {
     Main.keybindingManager.removeHotKey('overlay-key-' + this.instance_id);
+    global.settings.disconnect(this.panelEditModeChangedId);
```

Comment:
If using signalManager instead, this could be handled with the [disconnectAllSignals](https://github.com/claudiux/cinnamon-spices-applets/blob/d1d3ae27fa0303910ab0be06848bfebf55149986/Cinnamenu%40json/files/Cinnamenu%40json/applet.js#L2432) call.